server/autolinkclient/client.go

```diff
@@ -45,22 +49,79 @@ func (c *Client) Add(links ...autolink.Autolink) error {
 			return err
 		}
 
-		req, err := http.NewRequest("POST", "/"+autolinkPluginID+"/api/v1/link", bytes.NewReader(linkBytes))
+		resp, err := c.call("/"+autolinkPluginID+"/api/v1/link", http.MethodPost, linkBytes, nil)
```

Comment:
After the `c.call` we need to close the `resp`, as it was moved out of the inner function:

```go
resp, err := c.call("/"+autolinkPluginID+"/api/v1/link", http.MethodPost, linkBytes, nil)
if err != nil {
	return err
}
defer resp.Body.Close()
```